src/js/profile/mobile/widget/Calendar.js

```diff
@@ -110,22 +113,30 @@
 						day = day - 1;
 						leftDays = leftDays - 1;
 						cell = row.insertCell();
-						cell.innerHTML = prevLastDate - day;
-						cell.style.opacity = "10%";
+						cell.insertAdjacentHTML("afterbegin", divElement);
+						cell.firstChild.innerHTML = prevLastDate - day;
+						cell.firstChild.style.opacity = "10%";
 					}
 					while (leftDays != 0) {
 						if (setDays > prevLastDate) { // next disabled days.
 							cell = row.insertCell();
-							cell.innerHTML = nextMonthDate;
-							cell.style.opacity = "40%";
+							cell.insertAdjacentHTML("afterbegin", divElement);
+							cell.firstChild.innerHTML = nextMonthDate;
+							cell.firstChild.style.opacity = "40%";
 							leftDays = leftDays - 1;
 							nextMonthDate = nextMonthDate + 1;
 						} else { // current enabled days.
 							cell = row.insertCell();
-							cell.innerHTML = setDays;
-							cell.style.opacity = "100%";
+							cell.insertAdjacentHTML("afterbegin", divElement);
+							cell.firstChild.innerHTML = setDays;
+							cell.firstChild.style.opacity = "100%";
 							setDays = setDays + 1;
 							leftDays = leftDays - 1;
+
+							if (cell.firstChild.innerHTML === String(defaultToday)) { // today selected.
```

Comment:
Maybe better use `defaultToday.toLocaleString()`
or compare to `setDays`

Reply:
done.